src/drivers/kinetis/i2c.c

```diff
@@ -40,7 +40,8 @@ void i2c_init(i2c_config_t config, uint32_t clk_f_hz) {
     base->A1 = 0;
     base->F = 0;
     base->C1 = 0;
-    base->S = 0xFFu;
+    base->S = 0x80;
```

Comment:
What change does this make?

Reply:
apparently `0xFF` was not the reset value of the register and it set a bunch of things that we didn't intend on setting.